src/Robo/Plugin/Traits/SlackNotifierTrait.php

```diff
@@ -79,16 +74,13 @@ protected function sendSlackNotification(string $username, string $text): void
         try {
             $client = new Client(['timeout' => 5]);
             $client->post($slack_webhook_url, ['body' => json_encode($payload, JSON_THROW_ON_ERROR)]);
-        } catch (RequestException $exception) {
+        } catch (RequestException) {
```

Comment:
A question for here and other places, should we be doing anything with the exception variable? Logging it could be helpful? I realize that is outside the scope of this PR though.

Reply:
If there is data in there that is useful, then yes, we should. Since this is a CLI command and I am outputting an error, I didn't see a need to use any of the data in the exception.